File: app/src/main/java/com/keelim/nandadiagnosis/help/activity/OpenSourceActivity.java
package com.keelim.nandadiagnosis.help.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

import com.keelim.nandadiagnosis.R;

public class OpenSourceActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_open_source);

        Toast.makeText(this, "오픈소스 라이선스 메뉴 입니다.", Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/keelim/cnubus/activities/OpenSourceActivity.kt
package com.keelim.cnubus.activities

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.keelim.cnubus.R

class OpenSourceActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_open_source)
        Toast.makeText(this, "오픈소스 기능 고지 입니다.", Toast.LENGTH_LONG).show()
    }

}
